modules/addons/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "vpc_cni_id" {
+  description = "The id of the Amazon VPC CNI addon"
+  value       = aws_eks_addon.vpc_cni[0].id
```

Comment:
does it will be working when I will not create this addon (there will be no element 0)?

Reply:
Fixed the count=0 for all addons